packages/@snowpack/plugin-build-script/.npmignore

```diff
@@ -0,0 +1,2 @@
+tsconfig.json
+*.ts
```

Comment:
Here, and in `plugin-run-script`, we ignore built files from Git, but we ignore TS files from npm.